### PR TITLE
Addressing Issue Number 12

### PR DESCRIPTION
**Pull request by VarthanV**
I have solved the Issue number 12 that was opened by fetching the title and the description of the website when the extension is clicked under the guidance of @xlogix

### CODE REVIEW

File: src/content.js
Comment (arjunmahishi) [inline src/content.js]: we can just do `document.title` right?

File: src/content.js
Comment (arjunmahishi) [inline src/content.js]: we should send this data a json like:
```
{
    "title": " ",
    "description": " ",
    ...
}
```

File: src/popup/popup.js
Comment (arjunmahishi) [inline src/popup/popup.js]: where is this `message` element?

File: src/popup/popup.js
Comment (arjunmahishi) [inline src/popup/popup.js:19]: is this the only way to inject data into popup?
if so, let's name the script something else. Something more specific and descriptive.

File: src/auth/auth.js
Comment (arjunmahishi) [inline src/auth/auth.js]: remove this if this was just a test. let's keep auth.js clean

File: src/auth/auth.js
Comment (arjunmahishi) [inline src/auth/auth.js]: same as above

**Review by arjunmahishi — CHANGES_REQUESTED** [review]
Apart from the comments, get rid of unwanted new lines

**Comment by VarthanV** [comment]
I have made the necessary changes  and returned a json object to the main file

**Comment by VarthanV** [comment]
No Bro what Abhishek bro said is to extract the data and send it to the backend. I am just learning GraphQL the message is nothing but a div in auth.html to make sure that my logic works . Please specify me what  i have to do after extracting the data

File: src/popup/popup.js
Comment (arjunmahishi) [inline src/popup/popup.js]: This still not addressed. Can you point me to the "message" element?

File: src/popup/popup.js
Comment (arjunmahishi) [reply src/popup/popup.js:19]: Let's rename `content.js` to something like `metadata-fetcher.js` or something similar

**Review by arjunmahishi — CHANGES_REQUESTED** [review]

**Comment by arjunmahishi** [comment]
We can merge this once the "message" element is sorted out. (this is assuming the changes are tested and working fine)

**Comment by VarthanV** [comment]
Content.js is  a convention to name the file otherwise we cannot access the dom elements and get data . I have removed the message div I just used it for debugging the data will be returned as JSON to the popup file. If you want to know whether it works i have logged the title and metadata in the console when the site loads we can disable that after you approve the PR.

**Review by arjunmahishi — APPROVED** [review]

**Comment by arjunmahishi** [comment]
@VarthanV  Thank you for your contributions!

**Pull request merged**
